torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -256,10 +256,35 @@ NvrtcFunction nvrtcCompile(
   // Major and minor is determined by device properties and
   // possibly "downcompiled" to a lower (compatible) compute architecture
   // based on the NVRTC version
-  const int major = prop->major;
-  const int minor = prop->minor;
+  int major = prop->major;
+  int minor = prop->minor;
   nvrtcProgram program;
 
+  if (nvrtc_major <= 7 && prop->major > 5) { // 7 supports 2-5.x
+    major = 5;
+    minor = 0;
+  } else if (nvrtc_major <= 8 && prop->major > 6) { // 8 supports 2-6.x
+    major = 6;
+    minor = 0;
+  } else if (nvrtc_major <= 9 && prop->major >= 7) { // 9 supports 3-7.2
+    major = 7;
+    if (prop->major == 7 && prop->minor <= 2)
+      minor = prop->minor;
+    else
+      minor = 0;
```

Comment:
Why exception is made here, but in case of CUDA-8? (Which supports sm_61?)
Also, please use curly brackets even for a single line blocks or ternary
```suggestion
    minor =  (prop->major == 7 && prop->minor <= 2) ? minor: 0;
```

Reply:
> but in case of CUDA-8? (Which supports sm_61?)

Honestly I don't know the proper support of CUDA-8. I can double check that. But given how ancient cuda8 is, I'm not curious if we are eager to patch it.

Reply:
Pytorch no longer supports cuda 8, it's ok to remove all code related to cuda 8 and earlier.